src/m_misc.c

```diff
@@ -1952,21 +1952,21 @@ default_t defaults[] = {
   {
     "joy_sensitivity_turn",
     (config_t *) &joy_sensitivity_turn, NULL,
-    {50}, {0, 100}, number, ss_keys, wad_no,
+    {36}, {0, 100}, number, ss_keys, wad_no,
```

Comment:
Let's go with these settings for now and they can always be refined based on feedback before the next release.

Reply:
I like this, "extra turn sensitivity" is interesting 👍